common/src/main/java/com/wynntils/mc/mixin/PlayerTabOverlayMixin.java

```diff
@@ -18,11 +18,17 @@
 
 @Mixin(PlayerTabOverlay.class)
 public class PlayerTabOverlayMixin {
-    @Inject(method = "render", at = @At("HEAD"), cancellable = true)
-    private void render(PoseStack poseStack, int width, Scoreboard scoreboard, Objective objective, CallbackInfo ci) {
+    @Inject(
+            method =
+                    "render(Lnet/minecraft/client/gui/GuiGraphics;ILnet/minecraft/world/scores/Scoreboard;Lnet/minecraft/world/scores/Objective;)V",
+            at = @At("HEAD"),
+            cancellable = true)
+    private void render(
+            GuiGraphics guiGraphics, int width, Scoreboard scoreboard, Objective objective, CallbackInfo ci) {
         RenderEvent.Pre renderEvent =
-                new RenderEvent.Pre(poseStack, 0, McUtils.window(), RenderEvent.ElementType.PLAYER_TAB_LIST);
+                new RenderEvent.Pre(guiGraphics, 0, McUtils.window(), RenderEvent.ElementType.PLAYER_TAB_LIST);
         MixinHelper.post(renderEvent);
+
```

Comment:
why is there a change on this empty line?

Reply:
I added it for readability. Why is that a problem..?